website/src/pages/go.tsx

```diff
@@ -4,23 +4,23 @@ import * as React from 'react'
 import BlogPosts from '../components/BlogPosts'
 import Layout from '../components/Layout'
 export default class GoList extends React.Component<any, any> {
-    public componentDidMount(): void {
-        const script = document.createElement('script')
-        script.src = '//js.hsforms.net/forms/v2.js'
-        const hubspot = document.getElementById('gopherConForm')
-        hubspot.appendChild(script)
-        script.addEventListener('load', () => {
-            ;(window as any).hbspt.forms.create({
-                portalId: '2762526',
-                formId: '9cfe9111-fe45-45f5-873a-9f4925affdfd',
-                target: '#gopherConForm',
-            })
-        })
-    }
+    // public componentDidMount(): void {
+    //     const script = document.createElement('script')
+    //     script.src = '//js.hsforms.net/forms/v2.js'
+    //     const hubspot = document.getElementById('gopherConForm')
+    //     hubspot.appendChild(script)
+    //     script.addEventListener('load', () => {
+    //         ;(window as any).hbspt.forms.create({
+    //             portalId: '2762526',
+    //             formId: '9cfe9111-fe45-45f5-873a-9f4925affdfd',
+    //             target: '#gopherConForm',
+    //         })
+    //     })
+    // }
```

Comment:
Is there a reason this is commented out but not deleted?

Reply:
Because it will be re-used once (if) we host the liveblog for GopherCon again.

Reply:
I'll fix.